website/field-content/src/views/editor/add-block.js

```diff
@@ -162,6 +162,7 @@ let AddBlock = ({ editorState, editor, blocks }) => {
 							padding: 0,
 							margin: 0,
 							border: 'solid 1px #eaeaea',
+							zIndex: '9999999999999',
```

Comment:
🤦‍♂

Reply:
Sorry not sorry

Reply:
There is a bit of "whack a mole" with z-indexes at the moment. We should revisit this later